pkgs/by-name/cl/clever-tools/package.nix

```
@@ -4,41 +4,47 @@
   fetchFromGitHub,
   nodejs_18,
   installShellFiles,
+  makeWrapper,
   stdenv,
 }:
 
 buildNpmPackage rec {
   pname = "clever-tools";
 
-  version = "3.8.3";
+  version = "3.9.0";
 
   nodejs = nodejs_18;
 
   src = fetchFromGitHub {
     owner = "CleverCloud";
     repo = "clever-tools";
     rev = version;
-    hash = "sha256-70wyu8+Jb9kR5lIucBZG9UWIufMhsgMBMkT2ohGvE50=";
+    hash = "sha256-nSTcJIZO/CMliAYFUGu/oA+VdtONDPwyj6vCr5Ry6ac=";
   };
 
-  npmDepsHash = "sha256-LljwS6Rd/8WnGpxSHwCr87KWLaRR2i7sMdUuuprYiOE=";
+  npmDepsHash = "sha256-+3/zSsO5+s1MUome3CQ1p1tN3OtWp+XE9Z6GSdDiRh8=";
 
-  dontNpmBuild = true;
+  nativeBuildInputs = [
+    installShellFiles
+    makeWrapper
+  ];
 
-  nativeBuildInputs = [ installShellFiles ];
+  installPhase = ''
+    mkdir -p $out/bin $out/src
+    cp build/clever.cjs $out/src/clever.cjs
```

To avoid potential collisions, this should be something like `$out/lib/clever-tools/clever.cjs` instead:
```suggestion
    mkdir -p $out/bin $out/src
    cp build/clever.cjs $out/lib/clever-tools/clever.cjs
```

---

Indeed, thanks a lot, I've applied the fix :+1: